src/amr/load_balancing/load_balancer_details.hpp

```diff
@@ -0,0 +1,36 @@
+#ifndef PHARE_AMR_LOAD_BALANCER_LOAD_BALANCER_DETAILS_HPP
+#define PHARE_AMR_LOAD_BALANCER_LOAD_BALANCER_DETAILS_HPP
+
+#include <string>
+#include <cstdint>
+
+#include "core/logger.hpp"
+#include "initializer/data_provider.hpp"
+
+namespace PHARE::amr
+{
+struct LoadBalancerDetails
+{
+    bool const active    = false;
+    bool const automatic = false;
+    bool const on_init   = false;
+
+    std::size_t const every = 0;
+    std::string const mode;
+
+    double const tolerance = .05;
+
+    LoadBalancerDetails static FROM(initializer::PHAREDict const& dict)
+    {
+        return {cppdict::get_value(dict, "active", false),
+                cppdict::get_value(dict, "auto", false),
+                cppdict::get_value(dict, "on_init", false),
+                cppdict::get_value(dict, "every", std::size_t{0}),
+                cppdict::get_value(dict, "mode", std::string{"nppc"}),
+                cppdict::get_value(dict, "tol", 0.05)};
```

Comment:
why not using the `tolerance` constant?

Reply:
that's a const class variable with a default value, not a constexpr